scriptmodules/emulators/ti99sim.sh

```diff
@@ -36,7 +36,7 @@ function install_ti99sim() {
 function configure_ti99sim() {
     mkRomDir "ti99"
     moveConfigDir "$home/.ti99sim" "$md_conf_root/ti99/"
+    ln -sf "$biosdir/TI-994A.ctg" "$md_inst/TI-994A.ctg"
```

Comment:
it should at least function for now in the interim. I'm sure there will be another legacy code cleanup and we can address the inadequacies of this script at that point. In the meantime get better!